tests/integration/targets/junos_l3_interfaces/tests/netconf/junos_l3_interfaces.yml

```diff
@@ -0,0 +1,91 @@
+---
+- name: bootstrap interfaces
+  junos_l3_interfaces:
+    config:
+      - name: ge-1/0/0
+        ipv4:
+          - address: 192.168.100.1/24
+          - address: 10.200.16.20/24
+      - name: ge-2/0/0
+        ipv4:
+          - address: 192.168.100.2/24
+          - address: 10.200.16.21/24
+      - name: ge-3/0/0
+        ipv4:
+          - address: 192.168.100.3/24
+          - address: 10.200.16.22/24
+    state: replaced
+  register: result
+
+- assert:
+    that:
+      - result is changed
+      - "'<nc:address><nc:name>192.168.100.1/24</nc:name></nc:address>' in result.xml[0]"
+      - "'<nc:address><nc:name>10.200.16.20/24</nc:name></nc:address>' in result.xml[0]"
```

Comment:
The after key in result can be used in assert instead of fetching the xml config

Reply:
Maybe we can just use both, wdyt?